subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/ui/SubscriptionWebViewViewModel.kt

```diff
@@ -100,6 +103,11 @@ class SubscriptionWebViewViewModel @Inject constructor(
     private val _currentPurchaseViewState = MutableStateFlow(CurrentPurchaseViewState())
     val currentPurchaseViewState = _currentPurchaseViewState.asStateFlow()
 
+    data class ViewState(val rebrandingEnabled: Boolean = false)
```

Comment:
If `subscriptionRebrandingFeatureToggle.isSubscriptionRebrandingEnabled()` is main thread safe and there's already a cached value for you to consume, why not using it directly here?

Reply:
You can't use `subscriptionRebrandingFeatureToggle` in the default value because it's injected after the class is initialised, but default values are evaluated before that.